### ISSUE TITLE
API workbench complains with double include in DataTypes

### ISSUE THREAD

**Issue by superruzafa** [mulesoft/api-workbench#132]
In the API workbench IDE (v0.8.20) I have a problem with a custom DataType which uses another custom DataTypes as shown in the image below:

![api-workbench](https://cloud.githubusercontent.com/assets/231822/13393976/2acf047e-dee5-11e5-8097-eb6e19578cfb.png)

It seems what I'm doing is reasonably logic but the API Workbench complains.

Note: I'm using `!includes` here because the impossibility of make use of `uses` in `DataType` documents, as reported [here](http://forums.raml.org/t/include-library-in-datatype-definition/1187) and [here](https://github.com/raml-org/raml-spec/issues/226).

**Comment by ddenisenko** [comment]
Thank you for reporting, it is reproducible.

**Comment by sichvoge** [comment]
Yes, the error only turns up when using include on a `type` property.

**Comment by IsenGrim613** [comment]
Was trying to do type inheritance with DateType fragments and ran into this issue.

dataTypes/foo.raml
```
#%RAML 1.0 DataType
type: object
properties:
  id: string
  name: string
```

dataTypes/bar.raml
```
#%RAML 1.0 DataType
type: !include foo.raml
properties:
  additional_info: string
```

However, Atom highlights type: !include foo.raml as an error.
This is the error message:
```Syntax error:Expected "(" or name but end of input found. credential.ramlat line 2 col 1```

**Comment by sichvoge** [comment]
@IsenGrim613 Doing type inheritance with DataType was not a feature in RAML 1.0 RC1 that is being used by the API Workbench at the moment. Its been fixed in RC2 though.
